RawSpeed/DngDecoder.cpp

```diff
@@ -230,17 +234,23 @@ RawImage DngDecoder::decodeRawInternal() {
       } catch (TiffParserException) {
         ThrowRDE("DNG Decoder: Unsupported format, uncompressed with no strips.");
       }
-    } else if (compression == 7 || compression == 0x884c) {
+    } else if (compression == 7 || compression == 8 || compression == 0x884c) {
```

Comment:
Hmm, now that i think the second time, for now just do https://github.com/darktable-org/rawspeed/pull/17/files/4f2bdbbf76cb8c23be314d0b76effafd397f206e#r95054866